docs/getting_started.md

```diff
@@ -0,0 +1,142 @@
+# Datadog PHP integration
+
+Datadog's PHP APM integration provides the ability to trace critical parts of your PHP applications by automatically instrumenting code and submitting data to Datadog’s APM service using the OpenTracing API.
+
+To provide a great out-of-the-box experience that automatically instruments common libraries without requiring intrusive code changes or wrapping every object, we provide a PHP extension that allows introspecting any PHP function or method.
+
+## Datadog PHP extension installation
+
+Datadog’s PHP extension (`ddtrace`) allows introspection of arbitrary PHP code.
+
+At this moment it is only distributed in source code form, and requires manual compilation.
+
+### Compiling and installing the extension manually
+
+```bash
+mkdir dd-trace
+cd dd-trace
+curl -L https://github.com/DataDog/dd-trace-php/archive/v0.2.2.tar.gz | tar x --strip-components=1
```

Comment:
@pawelchcki and @chuck will also update this to `0.2.3` ack?

Reply:
v0.2.3 ;)

Reply:
```suggestion
curl -L https://github.com/DataDog/dd-trace-php/archive/v0.2.3.tar.gz | tar x --strip-components=1
```